DataFormats/TPC/src/DataFormatsTPCLinkDef.h

```diff
@@ -0,0 +1,24 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifdef __CLING__
+
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+#pragma link C++ class o2::DataFormat::TPC::ClusterHardware+;
```

Comment:
Remind me about what goes in the dictionary, @sawenzel can you help here? The new data structures do not have any `CLASS_DEF/CLASS_IMP` macros. Do we have to think about the `+` flag?